src/main/java/seedu/address/model/person/LoanRate.java

```diff
@@ -0,0 +1,52 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Loan's rate in the loan book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidRate}
+ */
+public class LoanRate {
+
+    public static final String LOANRATE_VALIDATION_REGEX = "([1-9]\\d*(\\.\\d*[1-9])?)|(0\\.\\d*[1-9])";
+    public static final String MESSAGE_LOANRATE_CONSTRAINTS =
+            "Rate should only contain numbers. It should be greater than 0 and it can have at most 2 decimal places";
+
+    public final String rate;
```

Comment:
I think we should store rate as a double. So that it is in a mathematical form...

Reply:
Because the value represents a rate (the amount of money per hour). It makes sense to store the rate as a double.

Reply:
I can do it when I integrate this data field into the DataField class.